ivy/functional/frontends/jax/random.py

```diff
@@ -45,3 +45,9 @@ def _get_seed(key):
 def beta(key, a, b, shape=None, dtype=None):
     seed = _get_seed(key)
     return ivy.beta(a, b, shape=shape, dtype=dtype, seed=seed)
+
+
+def dirichlet(key, alpha, shape=None, dtype="float32"):
```

Comment:
You need to add the `@handle_jax_dtype` and `@to_ivy_arrays_and_back` decorators to your function, like all the other functions in this file.